include/engine/api/table_api.hpp

```diff
@@ -116,13 +124,48 @@ class TableAPI final : public BaseAPI
                                {
                                    return util::json::Value(util::json::Null());
                                }
+                               // division by 10 below because the duration up until this
+                               // point in the code is in deciseconds (10s)
+                               // as duration is computed during contraction and stored as int
+                               // rather than float to save on memory usage
+                               // for the extract -> shortcut (CH/MLD) -> route pipeline
```

Comment:
I wonder if the comment is too long? Personally, the information that duration is deciseconds would suffice (so just the two first lines are enough).

Also maybe let's make this shorter 🙂

`division by 10 below because the duration up until this point in the code is in deciseconds (10s)`
->
`division by 10 because the duration is in deciseconds (10s)`